src/libtilt/filters/tests/test_b.py

```diff
@@ -0,0 +1,23 @@
+import torch
+
+from libtilt.filters import bfactor_2d
+
+def test_bfactor_2d():
+    # Generate an image
+    img1 = torch.zeros((1,4))
+    img2 = torch.ones((3,4))
+    img = torch.vstack((img1,img2))
```

Comment:
took me a little bit to understand what had been constructed, how about
```suggestion
    image = torch.zeros((4,4))
    image[1:, :] = 1
```